### DISCUSSION TITLE
proposal: Remove "databend" in tags

### DISCUSSION THREAD

**Discussion by Xuanwo** [databendlabs/databend#4685] [Ideas]
We have `A-databend-query`, `A-datebend-meta`, .. in tags that don't make sense. 

![image](https://user-images.githubusercontent.com/5351546/161490342-615d85a7-b62e-4609-8de3-81c67670269d.png)

Because our project is databend.

I propose to rename all those tags to `A-query`, `A-meta` without databend.

**Comment by bohutang** [comment]
Make sense 👍 
Renamed!
